react-client/src/Components/BookmarkPanel/AddBookmarkPanel.js

```diff
@@ -4,7 +4,7 @@ import { AiOutlineClose } from "react-icons/ai";
 
 const AddBookmarkPanel = ({ domType, elClassNames, saveBookmark, onClose, x, y, domId }) => (
   <div className='add__bookmark-panel' style={{ top: y + 270, left: x + 15 }}>
-    <span className='header'>
+    <span className='add__bookmark-header'>
```

Comment:
Incorrect BEM CSS class naming. Please refer here for proper naming convention of BEM: https://cssguidelin.es/#bem-like-naming